modules/sdk-lib-mpc/src/tss/index.ts

```diff
@@ -1 +1 @@
-export * as Ecdsa from './ecdsa';
+export * from './ecdsa';
```

Comment:
Are we only going to use this library for ecdsa, Named export can help to distinguish between similar types/ functions between Ecdsa and Eddsa, without it can be confusing

Reply:
We can also squash the commits